File: FoodDeliveryApi/Models/OrderDish.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace FoodDeliveryApi.Models
{
    public class OrderDish
    {
        public int Id { get; set; }
        public int DishId { get; set; }
        public Dish Dish { get; set; }
        public int OrderId { get; set; }
        public Order Order { get; set; }
        public int Count { get; set; }
    }
}

File: delivery-food-app/src/componets/OrderDish.js
import redux from 'redux'

function OrderDish({orderDish}) {
    return (
        <div className='order-dish border'>
            <p className='counter'>({orderDish.count})</p>
            <p>{orderDish.title}</p>
        </div>       
    )
}

export default OrderDish